python/cac_tripplanner/templates/base.html

```diff
@@ -69,6 +69,11 @@
 
     {% endblock %}
 
+    <script type="text/javascript">
```

Comment:
Shouldn't this block be in `map.html`, rather than the base page?  Looks like it's on the `map` view only.